src/docfx/docset/Document.cs

```diff
@@ -188,7 +189,7 @@ public static (Error error, Document doc) TryCreate(Docset docset, string path,
             Debug.Assert(!Path.IsPathRooted(path));
 
             var filePath = PathUtility.NormalizeFile(path);
-            var type = GetContentType(filePath);
+            var type = docset.Config.Extend.Contains(filePath, PathUtility.PathComparer) ? ContentType.Unknown : GetContentType(filePath);
```

Comment:
should we also exclude other configuration files like github user info and build info?